doc/seg-schema.json

```diff
@@ -1,152 +1,197 @@
+            {
+              "$ref": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/LO"
+            }
+          ],
+          "default": "dcmqi"
         },
-
-        "AnatomicRegionModifierSequence" : {
-          "$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/codeSequence",
-          "required" : false
+        "ClinicalTrialTimePointID": {
+          "allOf": [
+            {
+              "$ref": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/LO"
+            }
+          ],
+          "default": "1"
         },
-
-        "SegmentAlgorithmType": {
-          "allOf" : [
-            {"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/CS"},
-            {"enum" : ["AUTOMATIC","MANUAL","SEMIAUTOMATIC"]},
-            {"default" : "SEMIAUTOMATIC"}
+        "SeriesDescription": {
+          "allOf": [
+            {
+              "$ref": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/LO"
+            }
           ],
-          "required" : true
+          "default": "Segmentation"
         },
-
-        "SegmentAlgorithmName": {
-          "$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/LO"
+        "SeriesNumber": {
+          "allOf": [
+            {
+              "$ref": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/IS"
+            }
+          ],
+          "default": "300"
         },
-
-        "recommendedDisplayRGBValue": {
-          "type": "array",
-          "minItems" : 3,
-          "maxItems" : 3,
-          "items" : { "$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/US"},
-          "default" : ["128","174","128"],
-          "required" : false
+        "InstanceNumber": {
+          "allOf": [
+            {
+              "$ref": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/IS"
+            }
+          ],
+          "default": "1"
         },
-
-        "RecommendedDisplayCIELabValue": {
-          "type": "array",
-          "minItems" : 3,
-          "maxItems" : 3,
-          "items" : { "$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/US"},
-          "default" : ["43803","26565","37722"],
-          "required" : false
+        "BodyPartExamined": {
+          "allOf": [
+            {
+              "$ref": "https://raw.githubusercontent.com/qiicr/dcmqi/master/doc/common-schema.json#/definitions/CS"
+            }
+          ],
+          "default": ""
+        }
+      }
+    },
+    "segmentAttributes": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "SegmentedPropertyCategoryCodeSequence",
+          "SegmentedPropertyTypeCodeSequence",
+          "SegmentAlgorithmType"
+        ],
+        "properties": {
```

Comment:
@fedorov all required attributes moved into one array on top of properties, not for each separate